src/main/java/org/folio/circulation/domain/validation/RequestLoanValidator.java

```diff
@@ -37,4 +53,32 @@ public RequestLoanValidator(LoanRepository loanRepository) {
         return singleValidationError(new ValidationError(message, parameters));
       }).map(loan -> requestAndRelatedRecords));
   }
+
+  public CompletableFuture<Result<RequestAndRelatedRecords>> refuseWhenUserHasAlreadyBeenLoanedOneOfInstancesItems(
+    RequestAndRelatedRecords requestAndRelatedRecords) {
+
+    final Request request = requestAndRelatedRecords.getRequest();
+
+    return itemByInstanceIdFinder.getItemsByInstanceId(UUID.fromString(request.getInstanceId()))
+      .thenCombine(loanRepository.findOpenLoansByUserIdWithItem(LOANS_PAGE_LIMIT, request.getUserId()),
+        (items, loans) -> verifyNoMatchBetweenItemsAndLoans(items, loans, requestAndRelatedRecords));
+  }
+
+  private Result<RequestAndRelatedRecords> verifyNoMatchBetweenItemsAndLoans(Result<Collection<Item>> items,
+      Result<MultipleRecords<Loan>> loans, RequestAndRelatedRecords requestAndRelatedRecords) {
+    List<String> itemIds = items.value().stream()
+      .map(Item::getItemId)
+      .collect(Collectors.toList());
+
+    return loans.value().getRecords().stream()
+      .noneMatch(loan -> itemIds.contains(loan.getItemId()))
+      ? succeeded(requestAndRelatedRecords)
+      : failed(createValidationError(requestAndRelatedRecords));
+  }
+
+  private ValidationErrorFailure createValidationError(RequestAndRelatedRecords requestAndRelatedRecords) {
+    return new ValidationErrorFailure(
+      new ValidationError("This requester has some item of instance on loan.",
```

Comment:
"One of the items of the requested title is already loaned to the requester"

Reply:
done